src/test/java/com/google/devtools/build/lib/rules/objc/ObjcStarlarkTest.java

```diff
@@ -1498,9 +1498,10 @@ public void testDisallowSDKFrameworkAttribute() throws Exception {
     assertThat(e)
         .hasMessageThat()
         .contains(
-            "ERROR /workspace/examples/apple_starlark/BUILD:1:13: in sdk_frameworks attribute of"
-                + " objc_library rule //examples/apple_starlark:lib: sdk_frameworks attribute is"
-                + " disallowed. Use explicit dependencies instead.");
+            "ERROR /workspace/examples/apple_starlark/BUILD:1:13: "
+                + "in objc_library rule //examples/apple_starlark:lib:");
+
+    assertContainsEvent("sdk_frameworks attribute is disallowed. Use explicit dependencies instead.");
```

Comment:
Minor error text change with fail vs Java. Same behavior and I tried to nearly match it